packages/freezed/test/single_class_constructor_test.dart

```diff
@@ -172,6 +172,47 @@ void main() {
     expect('${SingleNamedCtor.named(42)}', 'SingleNamedCtor.named(a: 42)');
   });
 
+  test('single-case union does have map', () async {
+    await expectLater(compile(r'''
```

Comment:
Instead of compile/completes, use the `map` method normally.
The current test assert that there is a `map` member, but not what it is and what it does.